modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/Loader.scala

```diff
@@ -69,14 +70,18 @@ object Loader {
     val periodicMetrics: Stream[F, Unit] =
       Monitoring[F].periodicMetrics.report
 
-    val init: F[Unit] = TargetCheck.blockUntilReady[F, C](config.readyCheck, config.storage) *>
-      NoOperation.prepare(config.schedules.noOperation, control.makePaused) *>
-      Manifest.initialize[F, C](config.storage) *>
-      Transaction[F, C].transact(addLoadTstampColumn[C](config.storage))
+    val blockUntilReady = TargetCheck.blockUntilReady[F, C](config.readyCheck, config.storage) *>
+      Logging[F].info("Target check is completed")
```

Comment:
Are we missing a retry block around this blockUntilReady?  In #964 we said it was important that the initialization should be resilient to failures, but I think now it's going to crash if there's an error connecting to the warehouse.